docs/reference/announcements-release-notes/850/850-announcements.md

```diff
@@ -94,84 +94,3 @@ The number of replicas for the Web Modeler REST API and web app deployments can
 
 The Camunda Helm charts have been updated to use the new Bitnami Docker repository.
 See [Bitnami Docker repository migration](/self-managed/installation-methods/helm/upgrade/index.md#bitnami-docker-repository-migration) for migration details.
-
```

Comment:
I am assuming this can be removed? If not, I just need to update the links which route to 8.4 docs.